component/Badges/badge.css

```diff
@@ -44,7 +44,7 @@
     border-radius: 50%;
     cursor: pointer;
 }
-.status-badge{
+.avatar-status{
     position:absolute;
     width:14px;
```

Comment:
you can use rem here

Reply:
oh i forgot i had to change them thanks for pointing it out i will do it